src/backend/src/services/deadline-notifications.services.ts

```diff
@@ -0,0 +1,71 @@
+import prisma from '../prisma/prisma';
+import { TaskWithAssignees, getTeamFromTaskAssignees, usersToSlackPings } from '../utils/deadline-notifications.utils';
+import { sendMessage } from '../integrations/slack';
+
+export default class DeadlineNotificationsService {
+  /**
+   * Sends the task deadline slack notifications for all tasks with a deadline of the given date
+   * @param deadline the beginning of the deadline date (at 12am)
+   */
+  static async sendTaskDeadlineSlackNotifications(deadline: Date) {
+    const startOfDay = deadline;
```

Comment:
I'd honestly make two helper methods, one that returns startOfDayTomorrow, and one that returns endOfDayTomorrow, and then just call them in the service method